### DISCUSSION TITLE
利用这里的数据，我写了一个网站，在地图上显示楼盘数据

### DISCUSSION THREAD

**Discussion by ritajie** [WeNeedHome/SummaryOfLoanSuspension#1045] [Ideas]
### 为什么要写网站？
网站可以让更多人看到！
网站上线两周了，没有收到相关部门的下线要求。

### 现在完成得怎么样了？
仓库也已经收到了 100+ star，技术栈：Django + jQuery。
* 数据来源：此仓库 [WeNeedHome](https://github.com/WeNeedHome/SummaryOfLoanSuspension)
* 网页地址：https://building.lulaolu.com
* 代码仓库：https://github.com/ritajie/incomplete-projects

### 接下来的工作是什么，需要什么协助？
1. 需要持续更新数据，将 WeNeedHome 的数据呈现在网站上
2. 需要优化网站体验，包括 UI、缓存、更方便的功能
3. 需要换个域名（如果有必要的话），当前只是临时使用了我的个人域名

### 欢迎大家一起来贡献！！！
![alt text](https://github.com/ritajie/incomplete-projects/blob/master/incomplete_projects/static/img/demo.png?raw=true)

**Comment by ytx0574** [comment]
重庆阳城未来悦的坐标应该不对

**Reply by ritajie** [reply]
明天我来确认下

**Comment by yuhan6665** [comment]
很棒啊 在这里主页上放个链接到你的网站吧

**Reply by ritajie** [reply]
好的，明天我提个 mr😎

**Reply by ritajie** [reply]
@yuhan6665 PR https://github.com/WeNeedHome/SummaryOfLoanSuspension/pull/1061

**Comment by guyanyouyou** [comment]
打不开了，https://libs.baidu.com/jquery/1.9.0/jquery.js 这个引用报500错误

**Reply by ritajie** [reply]
不能复现，我这边一切正常

**Comment by mstao** [comment]
太卡了

**Comment by guangxuanliu** [comment]
在手机端，chrome浏览器中比较卡。电脑端无此现象。

**Comment by masami1912** [comment]
句容泰禾金尊府位置和图片不对

**Comment by lettermessenger** [comment]
这都是违规信息啊，你不害怕请你进去喝茶吗？

**Reply by yuhan6665** [reply]
楼主签名：這個世界會好的。

**Comment by 56823** [comment]
怎么停了，你受到威胁了吗

**Reply by seepine** [reply]
作者被请喝茶了
